server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanQuestions.ts

```diff
@@ -61,7 +62,24 @@ export default class ContingencyPlanQuestions implements TasklistPage {
 
   questions: ContingencyPlanQuestionsRecord = questions
 
-  constructor(public body: ContingencyPlanQuestionsBody, private readonly application: Application) {}
+  contingencyPlanPartnerNames: Array<string> | null | Record<string, string>
+
+  constructor(public body: ContingencyPlanQuestionsBody, private readonly application: Application) {
+    let contingencyPlanPartnersNames = retrieveOptionalQuestionResponseFromApplication(
+      application,
+      'further-considerations',
+      'contingency-plan-partners',
+      'partnerAgencyDetails',
+    )
+
+    if (Array.isArray(contingencyPlanPartnersNames)) {
```

Comment:
Could we set `contingencyPlanPartnersNames` to default to an empty array, so we don't have to do the `isArray` check?

Reply:
Yeah this can be tidied up a lot